src/plenoptic/data/fetch.py

```diff
@@ -5,56 +5,61 @@
-    'ssim_analysis.mat': OSF_TEMPLATE.format('ndtc7'),
-    'msssim_images.tar.gz': OSF_TEMPLATE.format('5fuba'),
-    'MAD_results.tar.gz': OSF_TEMPLATE.format('jwcsr'),
-    'portilla_simoncelli_matlab_test_vectors.tar.gz': OSF_TEMPLATE.format('qtn5y'),
-    'portilla_simoncelli_test_vectors.tar.gz': OSF_TEMPLATE.format('8r2gq'),
-    'portilla_simoncelli_images.tar.gz': OSF_TEMPLATE.format('eqr3t'),
-    'portilla_simoncelli_synthesize.npz': OSF_TEMPLATE.format('a7p9r'),
-    'portilla_simoncelli_synthesize_torch_v1.12.0.npz': OSF_TEMPLATE.format('gbv8e'),
-    'portilla_simoncelli_synthesize_gpu.npz': OSF_TEMPLATE.format('tn4y8'),
-    'portilla_simoncelli_scales.npz': OSF_TEMPLATE.format('xhwv3'),
-    'sample_images.tar.gz': OSF_TEMPLATE.format('6drmy'),
-    'test_images.tar.gz': OSF_TEMPLATE.format('au3b8'),
-    'tid2013.tar.gz': OSF_TEMPLATE.format('uscgv'),
-    'portilla_simoncelli_test_vectors_refactor.tar.gz': OSF_TEMPLATE.format('ca7qt'),
-    'portilla_simoncelli_synthesize_torch_v1.12.0_ps-refactor.npz': OSF_TEMPLATE.format('vmwzd'),
-    'portilla_simoncelli_synthesize_gpu_ps-refactor.npz': OSF_TEMPLATE.format('mqs6y'),
-    'portilla_simoncelli_scales_ps-refactor.npz': OSF_TEMPLATE.format('nvpr4'),
-    'portilla_simoncelli_synthesize_torch_v1.12.0_ps-refactor-2.npz': OSF_TEMPLATE.format('en8du'),
+    "plenoptic-test-files.tar.gz": OSF_TEMPLATE.format("q9kn8"),
+    "ssim_images.tar.gz": OSF_TEMPLATE.format("j65tw"),
+    "ssim_analysis.mat": OSF_TEMPLATE.format("ndtc7"),
+    "msssim_images.tar.gz": OSF_TEMPLATE.format("5fuba"),
+    "MAD_results.tar.gz": OSF_TEMPLATE.format("jwcsr"),
+    "portilla_simoncelli_matlab_test_vectors.tar.gz": OSF_TEMPLATE.format("qtn5y"),
+    "portilla_simoncelli_test_vectors.tar.gz": OSF_TEMPLATE.format("8r2gq"),
+    "portilla_simoncelli_images.tar.gz": OSF_TEMPLATE.format("eqr3t"),
+    "portilla_simoncelli_synthesize.npz": OSF_TEMPLATE.format("a7p9r"),
+    "portilla_simoncelli_synthesize_torch_v1.12.0.npz": OSF_TEMPLATE.format("gbv8e"),
+    "portilla_simoncelli_synthesize_gpu.npz": OSF_TEMPLATE.format("tn4y8"),
+    "portilla_simoncelli_scales.npz": OSF_TEMPLATE.format("xhwv3"),
+    "sample_images.tar.gz": OSF_TEMPLATE.format("6drmy"),
+    "test_images.tar.gz": OSF_TEMPLATE.format("au3b8"),
+    "tid2013.tar.gz": OSF_TEMPLATE.format("uscgv"),
+    "portilla_simoncelli_test_vectors_refactor.tar.gz": OSF_TEMPLATE.format("ca7qt"),
+    "portilla_simoncelli_synthesize_torch_v1.12.0_ps-refactor.npz": OSF_TEMPLATE.format(
+        "vmwzd"
+    ),
+    "portilla_simoncelli_synthesize_gpu_ps-refactor.npz": OSF_TEMPLATE.format("mqs6y"),
+    "portilla_simoncelli_scales_ps-refactor.npz": OSF_TEMPLATE.format("nvpr4"),
+    "portilla_simoncelli_synthesize_torch_v1.12.0_ps-refactor-2.npz": OSF_TEMPLATE.format(  # noqa: E501
+        "en8du"
+    ),
 }
 DOWNLOADABLE_FILES = list(REGISTRY_URLS.keys())
 
-import pathlib
-from typing import List
+# ignore E402
+import pathlib  # noqa: E402
```

Comment:
I think this can be fixed by moving all the imports above the constant definitions. There's no real reason why I did it in this order.